ONSdigital/eq-author-app
# Ear 1702 force password change

https://jira.ons.gov.uk/browse/EAR-1702

To Test

- Spin up the Author
- Try to sign in using either a password shorter that 8 chars and then a common password eg facebook
- Check the appropriate message appears.
- Check you can still sign in with a suitable strength password.
